modules/virt-installing-virtctl-client-web.adoc

```diff
@@ -3,22 +3,16 @@
 // virt/install/virt-installing-virtctl.adoc
 
 [id="virt-installing-virtctl-client-web_{context}"]
-= Installing the virtctl client from the web console
+= Installing the virtctl client using the web console
```

Comment:
As stated below, it is not for only web console. This section explains how to install downloaded virtctl client. I recommend to change this title.